File: atcoder/abc044A_tak_and_hotels.py
# Vicfred
# https://atcoder.jp/contests/abc044/tasks/abc044_a
# simulation
n = int(input())
k = int(input())
x = int(input())
y = int(input())

total = 0

for night in range(1,n+1):
    if night <= k:
        total += x
    else:
        total += y

print(total)

File: atcoder/abc044A_tak_and_hotels.nim
# unihernandez22
# https://atcoder.jp/contests/abc044/tasks/abc044_a
# implementation

import sequtils, strutils

var n, k, x, y: int
(n, k, x, y) = (1..4).mapIt(stdin.readLine.strip.parseInt)

if n > k:
  echo k*x + (n-k)*y
else:
  echo n*x
